controller/array_action/array_mediator_svc.py

```diff
@@ -140,6 +140,10 @@ def maximal_volume_size_in_bytes(self):
     def max_lun_retries(self):
         return 10
 
+    @classproperty
+    def default_object_prefix(self):
```

Comment:
please add a unit test that asserts `default_object_prefix` is not larger than `max_volume_prefix_length` and `max_snapshot_prefix_length`

Reply:
done